### PR TITLE
Feat/email flow

### LINKED ISSUE #698: Emails flow: lifebanks users
Dep: #684

Front-end: @steph222 
Backend: @leisterfrancisco

### LINKED ISSUE #699: Emails flow: sponsors users
Dep: #684

Front-end: @steph222 
Backend: @leisterfrancisco

### LINKED ISSUE #700: Emails flow: donors users
Dep: #684

### PR DESCRIPTION
**Pull request by leisterfrancisco**
Resolves #698 
Resolves #699
Resolves #700

### What does this PR do?
Create cron-job to send emails to user every month, year or any specific time to send them reports, new offers, sponsors or lifebanks.

### How should this be tested?
- Make run
- Check monthly, yearly or any specific time email inbox.

### CODE REVIEW

File: hapi/src/cron-jobs/monthly-transaction.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function sayHello() {
```
Comment (leisterfrancisco) [inline hapi/src/cron-jobs/monthly-transaction.js:1]: Testing purpose

Comment (kuronosec) [reply hapi/src/cron-jobs/monthly-transaction.js:1]: If the purpose of this PR is testing something, why not lowering the cronjob time from a month to something more visible, like 10 minutes?

**Review by kuronosec — CHANGES_REQUESTED** [review]

File: hapi/src/cron-jobs/monthly-transaction.js (hunk 1 above)
Comment (kuronosec) [reply hapi/src/cron-jobs/monthly-transaction.js:1]: @leisterfrancisco ups the montly-transaction name confused me :)

**Review by kuronosec — APPROVED** [review]
LGTM

File: kubernetes/email-cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  name: hello
+spec:
+  schedule: "*/1 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: hello
+              image: dev-lifebank-hapi
```
Comment (leisterfrancisco) [inline kubernetes/email-cronjob.yaml:13]: Only for test purpose

File: kubernetes/cronjob.yaml [hunk 3]
```diff
@@ -10,10 +10,9 @@ spec:
         spec:
           containers:
             - name: hello
-              image: dev-lifebank-hapi
-              imagePullPolicy: IfNotPresent
-              command:
+              image: leister/lifebank-hapi:latest
```
Comment (xavier506) [inline kubernetes/cronjob.yaml:13]: Use environmental vars for image 

`${DOCKER_REGISTRY}/${IMAGE_NAME_HAPI}:${VERSION}`

Comment (leisterfrancisco) [reply kubernetes/cronjob.yaml:13]: @xavier506 
I used my local hapi image only for test purposes, my local image has some new files that eoscostarica506 doesn't has, I soon as a finish the new hapi logic, I'm gonna change it and get in touch with you to make a clean docker image upload to eoscostarica506

**Review by steph222 — APPROVED** [review]

**Review by Edgar097 — APPROVED** [review]

**Pull request merged**